sympy/polys/rings.py

```diff
@@ -552,6 +552,19 @@ def add_gens(self, symbols):
         syms = set(self.symbols).union(set(symbols))
         return self.clone(symbols=list(syms))
 
+    def symmetric_poly(self, n):
+        """Return the symmetric poly of given degree over this ring's gens."""
```

Comment:
elementary symmetric polynomial

Reply:
Maybe I can save this for a subsequent PR?

I made the change, but it results in a merge conflict. That's because we currently have this situation:

1. I started the `galois-groups` topic branch.
2. I opened and merged PR #24541, which made improvements to rings.py (and introduced the sloppy docstring, which I agree should be corrected).
3. I copied the new rings.py into the `galois-group` topic branch, since it is needed here.

```
      o---o---o   <-- PR #24541 made changes to rings.py
     /         \
o---o---o---o---o---o master
 \               \
  \               \  <-- cherry-picked the new rings.py from the other PR, since it's needed here
   \               \
    o---o---o---o---o---o galois-groups
```

If I now resolve the competing line changes on GitHub, [the docs state](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) that this will cause the entire `master` branch to be merged into the `galois-group` branch. I generally find those kinds of merges to be messy, and think it would be better not to do it.

Reply:
Okay, the change is now made on master as well as here, hence no conflict.